Content.Server/GameObjects/Components/Recycling/RecyclerComponent.cs

```diff
@@ -61,7 +67,7 @@ private void Clean()
 
         private bool CanGib(IEntity entity)
         {
-            return entity.HasComponent<IBody>() && !_safe && Powered;
+            return entity.HasComponent<IDamageableComponent>() && !_safe && Powered;
```

Comment:
This should check for IBody and then you could skip the entity.HasComponent<IBody>() check in line 90 as you aren't using IDamageableComponent code anywhere that I can see with the recycler.

Reply:
Removed the IDamageableComponent check, but kept the HasComponent<IBody> in line 90 as it is necessary so if gibbing is not possible, the player is not recycled (deleted from existance) in the CanRecycle method.